.env

```diff
@@ -1,4 +1,4 @@
 POSTGRES_USER=superadminpostgresuser
```

Comment:
.env file should not have values in git, you can add it in git ignore if you need it.